<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>655</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>655</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	padding: 10px;
	border: 2px solid rgb(214, 214, 214);
	border-radius: 20px;
}

QLineEdit:hover{
	padding: 10px;
	border: 2px solid gray;
	border-radius: 20px;
}

QLineEdit:focus{
	padding: 10px;
	border: 2px solid gray;
	border-radius: 20px;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	padding: 10px;
	border: 2px solid rgb(214, 214, 214);
	border-radius: 20px;
}

QLineEdit:hover{
	padding: 10px;
	border: 2px solid gray;
	border-radius: 20px;
}

QLineEdit:focus{
	padding: 10px;
	border: 2px solid gray;
	border-radius: 20px;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>410</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: white;
	border: 1px solid #faae2b;
	border-radius: 20px;
}

QPushButton:hover{
	background-color: white;
	border: 2px solid #faae2b;
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>480</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Already have an account?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(3, 148, 252);</string>
    </property>
    <property name="text">
     <string>Login now</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Image/OIG-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>I agree to Eclahtee's terms</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	padding: 10px;
	border: 2px solid rgb(214, 214, 214);
	border-radius: 20px;
}

QLineEdit:hover{
	padding: 10px;
	border: 2px solid gray;
	border-radius: 20px;
}

QLineEdit:focus{
	padding: 10px;
	border: 2px solid gray;
	border-radius: 20px;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	padding: 10px;
	border: 2px solid rgb(214, 214, 214);
	border-radius: 20px;
}

QLineEdit:hover{
	padding: 10px;
	border: 2px solid gray;
	border-radius: 20px;
}

QLineEdit:focus{
	padding: 10px;
	border: 2px solid gray;
	border-radius: 20px;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Buá»•i 6/resource2.qrc"/>
 </resources>
 <connections/>
</ui>
